boards/arm/cy8cproto_062_4343w/doc/index.rst

```diff
@@ -0,0 +1,205 @@
+
+- `PSoC 62 MCU SoC Website`_
+- `PSoC 62 MCU Datasheet`_
+- `PSoC 62 MCU Architecture Reference Manual`_
+- `PSoC 62 MCU Register Reference Manual`_
+- `CY8CPROTO-062-4343W PSoC 6 Wi-Fi BT Website`_
+- `CY8CPROTO-062-4343W PSoC 6 Wi-Fi BT User Guide`_
+- `CY8CPROTO-062-4343W PSoC 6 Wi-Fi BT Schematics`_
+
+Kit Features:
+==================
+- Support of up to 2MB Flash and 1MB SRAM
+- Dedicated SDHC to interface with WICED wireless devices.
+- Delivers dual-cores, with a 150-MHz Arm Cortex-M4 as the primary
+  application processor and a 100-MHz Arm Cortex-M0+ as the secondary
+  processor for low-power operations.
+- Supports Full-Speed USB, capacitive-sensing with CAPSENSE, a PDM-PCM
+  digital microphone interface, a Quad-SPI interface, 13 serial communication
+  blocks, 7 programmable analog blocks, and 56 programmable digital blocks.
+
+Kit Contents:
+==================
+- PSoC 6 Wi-Fi BT Prototyping Board
+- USB Type-A to Micro-B cable
+- Quick Start Guide
+
+Supported Features
+==================
+
+The board configuration supports the following hardware features:
+
++-----------+------------+-----------------------+
+| Interface | Controller | Driver/Component      |
++===========+============+=======================+
+| NVIC      | on-chip    | nested vectored       |
+|           |            | interrupt controller  |
++-----------+------------+-----------------------+
+| SYSTICK   | on-chip    | system clock          |
++-----------+------------+-----------------------+
+| GPIO      | on-chip    | GPIO                  |
++-----------+------------+-----------------------+
+| UART      | on-chip    | serial port-polling;  |
+|           |            | serial port-interrupt |
++-----------+------------+-----------------------+
+
+
+The default configuration can be found in the Kconfig
+
+:zephyr_file:`boards/arm/cy8cproto_062_4343w/cy8cproto_062_4343w_defconfig`
+
+
+System Clock
+============
+
+The PSoC 62 MCU SoC is configured to use the internal IMO+FLL as a source for
+the system clock. CM0+ works at 50MHz, CM4 - at 100MHz. Other sources for the
+system clock are provided in the SOC, depending on your system requirements.
+
+
+Install Infineon Connectivity Zephyr module
+*******************************************
+This modules includes assets which has any binary files (e.g WIFI/Bluetooth chip firmware,
+CM0p prebuilt images, etc).
+
+The following steps shows how to enable Infineon Connectivity Zephyr module in project:
+
+1. Get the Zephyr source code
+
+.. code-block:: console
+
+    west init
+
+2. Add an infineon_connectivity.yaml manifest file in to zephyr/submanifests folder. There is an example yaml file in this directory.
+Use the snippet of infineon_connectivity.yaml below:
+
+.. code-block:: yaml
+
+   manifest:
+     projects:
+       - name: infineon_zephyr_connectivity
+          url: https://github.com/Infineon/infineon_zephyr_connectivity.git
```

Comment:
Is this link functional?

Reply:
not for now.

for now you can use: https://github.com/ifyall/infineon_zephyr_connectivity
we had plan for https://github.com/Infineon/infineon_zephyr_connectivity.git make public, in scope of this PR..